src/Layout/Schema/index.style.js

```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
   width: 3rem;
   margin-left: 1.3rem;
 `;
+
```

Comment:
in Component  ```NextText``` how you will pass the on? function(onClick or onchange) and update your state if you created it as paragraph